tests/cases/fourslash/inlayHintsShouldWork22.ts

```diff
@@ -1,7 +0,0 @@
-/// <reference path="fourslash.ts" />
-
-////const a = "I'm very very very very very very very very very long";

```

Comment:
Is this one still captured somewhere?

Reply:
Nope. I removed this test because I've removed the truncation limit (for now) and I'm not sure how I'll handle it with interactive hints yet. 
Although this particular test is weird because it doesn't even generate inlay hints (because of the `const`) so I'm not even sure what functionality is testing.